pkg/distributors/contracts/RewardsScheduler.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/SafeERC20.sol";
+import "./interfaces/IMultiRewards.sol";
+
+// solhint-disable not-rely-on-time
+
+/**
+ * Scheduler for MultiRewards contract
+ */
+contract RewardsScheduler {
+    using SafeERC20 for IERC20;
+
+    IMultiRewards public multirewards;
+
+    constructor(IMultiRewards _multirewards) {
+        multirewards = _multirewards;
+    }
+
+    struct ScheduledReward {
+        IERC20 pool;
```

Comment:
`pool` is used as the parameter for the staking tokens everywhere in `multirewards`.  While there is no explicit check, since we for example, use the pool address to get the asset managers for the purpose of the allowlist, I think it is better to scope the use of this contract, and multirewards, to just staking pool bpt.

Reply:
It seems like `notifyReward` has `stakingToken` in its docs? That's where my confusion came from.

I personally don't mind either name (thought I don't think it's obvious that the Pool's contract is also the token contract), but we should at least use the same one everywhere.

Reply:
> It seems like notifyReward has stakingToken in its docs? That's where my confusion came from.

Sorry, where do you see `stakingToken`?  I see 2 references to staked token.  Also note multireward docs have been somewhat updated by the other PR for `Exiter`s that I opened last week.

Recommend we continue using `pool` everywhere